config/routes.rb

```diff
@@ -12,7 +12,6 @@
              }
 
   devise_scope :user do
```

Comment:
do we need this block at all?

Reply:
not in scope

Reply:
it is if its part of the facebook integration code

Reply:
i commented it out and put in a note that this is part of the omni auth system